### DISCUSSION TITLE
Multiple Home Assistant Instance backup scenario

### DISCUSSION THREAD

**Discussion by burghs** [lavinir/hassio-onedrive-backup#14] [Ideas]
Hi there, 

I have 3 different Home Assistant instances which I would like to backup to the same OneDrive account with its own configuration settings but it seems they conflict when I try. 
For example the HA test environment I don't need to keep more than one online backup per week so have the configuration settings set accordingly but production system I would have a different strategy which would involve more backups and longer retention but it appears that the backup settings from one is overwriting the other. Would this be a feature request or is there a way to achieve this by perhaps separating the folders where each back up to so that retention settings work for each as configured ?

**Comment by lavinir** [comment]
Hi,

Currently it treats the backups in OneDrive as a "single source of truth" and doesn't take into account multiple sources for the same account. I will add this request to the backlog.

**Comment by burghs** [comment]
Great, thank you

**Comment by BrettEBowman** [comment]
I've got the same situation. It would be great if you could use the Backup_name from the configuration to keep the online backups separate.

**Comment by flyguy62n** [comment]
Love the add-on and likewise I have more than one HA instance to backup.  +1 for this feature. (would it help to give it a +100?  1,000?

**Comment by lavinir** [comment]
Not necessary :) This is just used for me to prioritize upcoming features. This is definitely on the roadmap, stay tuned.

**Comment by lavinir** [comment]
This is now available in version 1.3
